library/google-services.json

```diff
@@ -1,33 +1,25 @@
 {
   "project_info": {
-    "project_number": "",
-    "firebase_url": "",
-    "project_id": "",
-    "storage_bucket": ""
+    "project_number": "316052285104",
```

Comment:
Can we just put some actual dummy values then?  I know there's nothing bad about leaking yours but it feels a little odd...